package.json

```diff
@@ -88,7 +88,7 @@
     "multiselect": "~0.9.12",
     "number_helpers": "^0.1.1",
     "patternfly": "^3.58.0",
-    "patternfly-react": "^2.25.5",
+    "patternfly-react": "^2.29.0",
```

Comment:
i assume we either merge this or https://github.com/theforeman/foreman/pull/6379 ?

Reply:
@ohadlevy notice #6379 updates `lodash` and `uuid` as well.

Reply:
@glekner please rebase now that #6379 is merged.

Reply:
done